core/client/fs/src/main/java/alluxio/client/block/stream/BlockInStream.java

```diff
@@ -61,7 +61,7 @@
   private PacketReader mPacketReader;
   private PacketReader.Factory mPacketReaderFactory;
 
-  private boolean mClosed = false;
+  protected boolean mClosed = false;
```

Comment:
Can you change this back to private? You can override the close method to store a boolean in the test class.

Reply:
done